src/bittrade/data/google_client.py

```diff
@@ -0,0 +1,101 @@
+"""Module to retrieves Google Finance data.
```

Comment:
The below doc string is for a single .py file that shows progress and plan see [here](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)